lib/defaultMetrics.js

```diff
@@ -28,6 +28,9 @@ var metrics = {
 };
 
 var existingInterval = null;
+// This is used to ensure the program throws on duplicate metrics during first run
+// We might want to consider not supporting running the default metrics function more than once
```

Comment:
https://github.com/prometheus/client_java/blob/master/simpleclient_hotspot/src/main/java/io/prometheus/client/hotspot/DefaultExports.java

Java noops. I don't see the use case we want to support (blacklisting certain metrics) as valid. Also, we should probably move collecting the metrics behind and explicit method call. In which case we can keep blacklisting, although I don't think it's necessary.